nomad/.config/openbox/menu.xml

```diff
@@ -146,6 +146,11 @@
 			<command>obconf </command>
 		</action>
 	</item>
+  <item icon="/usr/local/share/icons/hicolor/16x16/apps/plank.svg" label="Plank Preferences" comment="--- auto ---">
```

Comment:
The `comment="--- auto ---" ` is added by `update_obmenu` which deletes all entries with that comment before regenerating the menu. Instead of adding an entry to `menu.xml`, you could add a `plank-preferences.desktop` file to `config/usr/local/share/applications`.

Reply:
I do it.